src/examples/chrome/ChromeNavigation.tsx

```diff
@@ -74,7 +74,7 @@ const Example = () => {
         </NavItem>
       </Nav>
       {showSubNav && (
-        <SubNav>
+        <SubNav aria-label="chrome navigation example sub-nav">
```

Comment:
```suggestion
        <SubNav aria-label="chrome navigation example subnav">
```

[nit] the term "subnav" was adopted and used throughout the existing documentation and prop sheets